src/helpers/build.ts

```diff
@@ -28,6 +28,9 @@ export const buildStorybook = (
 };
 
 export const getBuildDir = (buildDir?: string): string => {
+  if (buildDir?.startsWith('/')) {
```

Comment:
if this intented to check for absolute path we can use `path.isAbsolute(buildDir)`

```suggestion
  if (!buildDir) return DEFAULT_BUILD_DIR
  if (path.isAbsolute(buildDir)) return buildDir
```